Sources/Core/AWSSDKHTTPAuth/AWSSigV4Signer.swift

```diff
@@ -5,33 +5,52 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-import Smithy
-import SmithyIdentityAPI
-import SmithyHTTPAPI
-import SmithyHTTPAuthAPI
-import AwsCommonRuntimeKit
-import ClientRuntime
-import Foundation
+import class AwsCommonRuntimeKit.HTTPRequestBase
+import class AwsCommonRuntimeKit.Signer
+import class SmithyHTTPAPI.SdkHttpRequest
+import class SmithyHTTPAPI.SdkHttpRequestBuilder
+import enum AwsCommonRuntimeKit.CommonRunTimeError
+import enum Smithy.ClientError
+import enum SmithyChecksumsAPI.ChecksumAlgorithm
```

Comment:
Merge conflict was not resolved properly as this was added back as a dependency

Reply:
Great catch, removed the import statement.